clients/drcachesim/simulator/caching_device_block.h

```diff
@@ -53,7 +53,7 @@ class caching_device_block_t {
     // where for the current replacement code we also set the counter at that time.
     caching_device_block_t()
         : tag_(TAG_INVALID)
-        , counter_(0)
+        , counter_(1)
```

Comment:
This changes the initial value for all cache types, though, and could break some other type.  Best to only change for LRU.

Reply:
Yes. I didn't notice this is a basic class. I have pushed new code.